### ISSUE TITLE
Too much fog/shroud on levels where AI sides fight one another

### ISSUE THREAD

**Issue by cooljeanius** [knyghtmare/The_Eight_of_Cembulad#10]
If AI sides are going to be fighting one another, I want to be able to *see* them doing so!

**Comment by knyghtmare** [comment]
Might be needing some checking on the code and see if any sighted events depend on it

**Comment by cooljeanius** [comment]
true... maybe instead of getting rid of the fog/shroud, instead just let you share vision with your allies more often?
